pasqal_cloud/__init__.py

```diff
@@ -127,7 +128,8 @@ def create_batch(
         self,
         serialized_sequence: str,
         jobs: List[CreateJob],
-        complete: bool = True,
+        complete: Optional[bool] = None,
+        open: Optional[bool] = None,
```

Comment:
Or even if we retrained the default value, because we set it to True as the parameter before, meaning it would need to be False and then we just raise a warning